### DISCUSSION TITLE
Error 502

### DISCUSSION THREAD

**Discussion by BrahianVT** [deta/cloud-docs#506] [Help]
here with my app in deta  URL: https://xyck4b.deta.dev/docs
 I'm getting the 502 bad gateway and it never reaches the 10s for the timeout, I
 tried all the options described here https://docs.deta.sh/docs/micros/visor but neither in the webpage with Visor nor with  **deta logs | deta logs --follow**  , it  didn't shows nothing  is all empty, any advice?

**Comment by AyushSehrawat** [comment]
https://basic.deta.dev/faq/micros/#im-getting-http-error-502-bad-gateway


Are you sure you are following all these, and not serving any content bigger than 5.5mb?

**Reply by BrahianVT** [reply]
here how to know if the size is bigger?  I guess  building Python project including dependencies in the same package

**Reply by Sokhavuth** [reply]
Make sure you update the requirements.txt file by doing:
```python
pip freeze > requirements.txt
```

**Reply by AyushSehrawat** [reply]
From 5.5mb i meant size of some image or file you are serving on your webserver

**Comment by BrahianVT** [comment]
I also tried this https://gist.github.com/jnsougata/e835e6c914278853af35331b28bdf163  and also it didn't work
